lib/menu.rb

```diff
@@ -0,0 +1,25 @@
+class Menu
+
+
+def initialize
+	@menu = Hash.new
+	@menu = {:beef_burger => 4.5,
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)